app/controllers/reports_controller.rb

```diff
@@ -21,6 +21,9 @@ def index
   def show
     @products = @report.user.products.not_wip.order(published_at: :desc)
     @recent_reports = Report.list.where(user_id: @report.user.id).limit(10)
+    Footprint.create_on_resource(@report, current_user) unless @report.user == current_user
+    @footprints = Footprint.fetch_for_resource(@report)
+    @footprint_total_count = Footprint.count_for_resource(@report)
```

Comment:
```suggestion
    @footprint_total_count = @footprints.count
```

これかもしくは `count`用のインスタンス変数は用意しなくても良いかもです。

Reply:
`@footprint_total_count `を使用していた`app/views/api/footprints/_footprint.json.jbuilder`と`app/views/api/footprints/index.json.jbuilder`は非Vue化で必要がなくなりましたので、インスタンス変数は用意しないようにしました